app/controllers/users.js

```diff
@@ -24,6 +25,52 @@ class UsersController extends BaseController {
     this.sessionRepository = new SessionRepository(ACTIVE);
   }
 
+  async #matched(data) {
```

Comment:
let's move this function to some other place.
Cause it duplicates wheat we already have in Contacts controller.

We can move to repository level